### PR TITLE
Remove SB parent from master

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

**Comment by geoand** [comment]
Work has not been completed yet, but I'm opening this to get the ball rolling on testing and reviews

File: pom.xml [hunk 1]
```diff
@@ -52,28 +80,138 @@
     </dependencies>
   </dependencyManagement>
 
+  <dependencies>
+    <dependency>
+      <groupId>com.jayway.restassured</groupId>
+      <artifactId>rest-assured</artifactId>
+      <version>2.9.0</version>
```
Comment (gytis) [inline pom.xml]: Maybe move version to property?

File: pom.xml [hunk 2]
```diff
@@ -52,28 +80,138 @@
...
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.jayway.awaitility</groupId>
+      <artifactId>awaitility</artifactId>
+      <version>1.7.0</version>
```
Comment (gytis) [inline pom.xml]: Maybe move version to property?

File: pom.xml [hunk 3]
```diff
@@ -52,28 +80,138 @@
...
+      <plugin>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-maven-plugin</artifactId>
+      </plugin>
+    </plugins>
+  </build>
+
   <profiles>
+
+  <profile>
```
Comment (gytis) [inline pom.xml]: We don't need this profile any more.

File: pom.xml [hunk 4]
```diff
@@ -52,28 +80,138 @@
...
+    </dependency>
+  </dependencies>
+
+  <modules>
+    <module>greeting-service</module>
+    <module>name-service</module>
+    <module>tests</module>
+  </modules>
+
+
```
Comment (gytis) [inline pom.xml]: Too many new lines?

File: pom.xml [hunk 5]
```diff
@@ -19,31 +19,59 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>io.openshift.booster</groupId>
-    <artifactId>spring-boot-booster-parent</artifactId>
-    <version>1.5.13-2</version>
+    <groupId>io.openshift</groupId>
```
Comment (gytis) [inline pom.xml:22]: I thought the idea was to get rid of all the parents?

**Review by gytis — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 4 above)
Comment (geoand) [reply pom.xml]: Stuff like this happened because I updated all the boosters in an automated fashion.

File: pom.xml (hunk 1 above)
Comment (geoand) [reply pom.xml]: Good idea, I just copied this from the parent, but it's probably best we put in a property

File: pom.xml (hunk 3 above)
Comment (geoand) [reply pom.xml]: Cool, I will remove it from all boosters then

File: pom.xml (hunk 5 above)
Comment (geoand) [reply pom.xml:22]: That's the final goal. @metacosm and I agreed to take it one step at a time

**Comment by geoand** [comment]
@gytis I addressed your comments

**Review by gytis — APPROVED** [review]

**Comment by geoand** [comment]
retest this please

**Pull request merged**